package.json

```diff
@@ -107,6 +107,7 @@
 	"dependencies": {
 		"@woocommerce/components": "3.2.0",
 		"gridicons": "3.3.1",
+		"react-number-format": "^4.2.0",
```

Comment:
I wonder if we could achieve the same results with [`@woocommerce/number`](https://www.npmjs.com/package/@woocommerce/number), so we use a dependency that we control.

Reply:
Last I checked it added extra dependencies we wish to avoid (lodash)

Reply:
Good point. I created a PR to remove lodash as a dependency (https://github.com/woocommerce/woocommerce-admin/pull/2977) but I'm ok using `react-number-format` anyway.